popgen/MCMC_Selection.go

```diff
@@ -82,15 +84,25 @@ func InitializeTheta(m float64, s float64, k int) Theta {
 
 //MetropolisHastings implements the MH algorithm for Markov Chain Monte Carlo approximation of the posterior distribution for selection based on an input allele frequency spectrum.
 //muZero and sigmaZero represent the starting hyperparameter values.
-func MetropolisHastings(data AFS, muZero float64, sigmaZero float64, iterations int, logSpace bool) ([]float64, []float64, []bool) {
+func MetropolisHastings(data AFS, muZero float64, sigmaZero float64, iterations int) ([]float64, []float64, []bool) {
+	//profiling test code
+	f, err := os.Create("testProfile.prof")
```

Comment:
This should probably be commented out in the "main" branch with your "DEBUG" syntax.